internal/services/network/web_application_firewall_policy_resource_test.go

```diff
@@ -111,6 +111,7 @@ func TestAccWebApplicationFirewallPolicy_complete(t *testing.T) {
 				check.That(data.ResourceName).Key("policy_settings.0.enabled").HasValue("true"),
 				check.That(data.ResourceName).Key("policy_settings.0.mode").HasValue("Prevention"),
 				check.That(data.ResourceName).Key("policy_settings.0.request_body_check").HasValue("true"),
+				check.That(data.ResourceName).Key("policy_settings.0.request_body_enforcement").HasValue("true"),
```

Comment:
Since this property has a default of `true`, we should update this test config to set it to `false`
```suggestion
				check.That(data.ResourceName).Key("policy_settings.0.request_body_enforcement").HasValue("false"),
```